components/TimePeriod/index.js

```diff
@@ -7,12 +7,7 @@ export default function TimePeriod({ timePeriod }) {
       <DateIcon />
       <time>
         {timePeriod.map(
-          (timePeriod) =>
-            `${
-              !timePeriod.endDate
-                ? `${timePeriod.startDate}`
-                : `${timePeriod.startDate} - ${timePeriod.endDate}`
-            }`
+          (timePeriod) => `${timePeriod.startDate} - ${timePeriod.endDate}`
```

Comment:
Now you are not handling the case when no `endDate` is passed :/

Possible solution - use a good ol' function!

```js
function formatTimePeriod({ startDate, endDate }) {
  if (endDate) {
    return `${startDate} - ${endDate}`;
  }
  return startDate;
}

export default function TimePeriod({ timePeriod }) {
  return (
    <DateContainer>
      <DateIcon />
      <time>
        {timePeriod.map(formatTimePeriod)}
      </time>
    </DateContainer>
  );
}
```